contrib/idxd-config-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+## Utility library for controlling and configuring Intel® In-Memory Analytics Accelerator (Intel® IAA).
+## This is to build libaccel-config which is only required by libqpl (./contrib/qpl).Hence, If ENABLE_QPL=False, this module should not be build as well.
+if (NOT ENABLE_QPL)
```

Comment:
Sorry for yet-another-comment but thinking about it a bit more, it I guess it would make the most sense if the CMake build descriptions of libaccel and QPL were in the same file (--> i.e. in contrib/qpl-cmake/CMakeLists.txt).

Reasons for this are:
- libaccel-config is required (as of now and in foreseeable future) only by QPL, so having both build from the same fill will be more self-contained
- Replicating l. 3-6 (this file) and l. 2-11 (qpl-cmake/CMakeLists.txt) is a bit fragile, even with a comment. For example, variable `ENABLE_QPL` (l. 3) will only be available if the QPL CMake file was processed before this file.

Therefore, would it be possible to move everything in this file into contrib/qpl-cmake/CMakeLists.txt? (please then also move the config.h/uuid.h headers into qpl-cmake). Thanks a lot!

Reply:
It does make sense.